test/integration/auth_test.go

```diff
@@ -1067,14 +970,8 @@ func TestKindAuthorization(t *testing.T) {
 	// This file has alice and bob in it.
 
 	// Set up a master
-	etcdStorage, err := framework.NewEtcdStorage()
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	expEtcdStorage, err := framework.NewExtensionsEtcdStorage(nil)
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
+	etcdStorage := framework.NewEtcdStorage()
```

Comment:
It seems that this one (and the following lines, e.g. the whole storageDestinations-related code) is no longer needed.
Can you please remove it?